packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -2204,7 +2207,7 @@ describeCompat(
 			},
 		);
 
-		itExpects(
+		(provider?.driver.type === "local" ? itExpects : itExpects.skip)(
```

Comment:
More common way is to do `if (provider.driver.type !== "local") { this.skip(); }` inside the test fn.  This looks fine too, I'll have it fixed up soon.  JFYI if you make more changes like this.